dev/release/000-run-docker.sh

```diff
@@ -47,10 +47,10 @@ else # boot2docker uid and gid
   GROUP_ID=50
 fi
 
-docker build -t "${IMAGE_NAME}-${USER_NAME}" - <<UserSpecificDocker
-FROM --platform=linux/amd64 ${IMAGE_NAME}
-RUN groupadd --non-unique -g ${GROUP_ID} ${USER_NAME} && \
-  useradd -l -g ${GROUP_ID} -u ${USER_ID} -k /root -m ${USER_NAME} && \
+docker buildx build --platform linux/amd64 -t "${IMAGE_NAME}-${USER_NAME}" - <<UserSpecificDocker
+FROM ${IMAGE_NAME}
```

Comment:
Do we need to specify the platform in FROM?

Reply:
@hangc0276 It's no needed, because we specify it when build.

Test on my M1 Mac
```
akka@cad8239aec01:~/code/java/bookkeeper-4-17$ arch
x86_64
```